src/deadline/job_attachments/models.py

```diff
@@ -215,6 +219,20 @@ class JobAttachmentS3Settings:
     # The S3 bucket prefix all files are stored relative to. (required)
     rootPrefix: str  # pylint: disable=invalid-name
 
+    @staticmethod
+    def from_root_path(root_path: str) -> JobAttachmentS3Settings:
+        path_split: list = root_path.split("/")
```

Comment:
I think this is the bug we chatted about :)

Reply:
I fixed it already, the buggy line was 226 where I had `len(path_split) != 2` condition (wrongly) assuming no additional `/` in root prefix.